packages/typespec-test/test/eventgrid_modular/spec/main.tsp

```diff
@@ -178,7 +178,7 @@ namespace Azure.Messaging.EventGrid {
     @doc("Single Cloud Event being published.")
     @body 
     event: {event: CloudEvent};
-  }, PublishResult>;
+  }, { @body _: PublishResult; }>;
```

Comment:
@timotheeguerin I believe here we should use `@body` not `@bodyRoot` because the PublishResult is an empty model.

tsp in spec repo: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/eventgrid/Azure.Messaging.EventGrid/main.tsp#L250

Reply:
we'll fix it in the spec repo when the issue I linked above is resolved.